### PR TITLE
Add support for grpc propagation

### PR DESCRIPTION
**Pull request by andremarianiello**
Closes https://github.com/opentracing-contrib/nginx-opentracing/issues/66

Adds the `opentracing_grpc_propagate_context` directive.

I extended the pattern laid out by `opentracing_propagate_context` and `opentracing_fastcgi_propagate_context` to create `opentracing_grpc_propagate_context`.

Still needs tests, but let me know if I am on the wrong track here.

### CODE REVIEW

**Comment by rnburn** [comment]
Yep, looks good. Thank you.

**Comment by andremarianiello** [comment]
@rnburn I added a case to test the opentracing_grpc_propagate_context directive. Let me know if this is adequate.

File: opentracing/src/opentracing_directive.h [hunk 1]
```diff
@@ -21,6 +21,13 @@ char *propagate_fastcgi_opentracing_context(ngx_conf_t *cf,
                                             ngx_command_t *command,
                                             void *conf) noexcept;
 
+//------------------------------------------------------------------------------
+// propagate_grpc_opentracing_context
+//------------------------------------------------------------------------------
+char *propagate_grpc_opentracing_context(ngx_conf_t *cf,
+                                            ngx_command_t *command,
+                                            void *conf) noexcept;
+
```
Comment (rnburn) [inline opentracing/src/opentracing_directive.h]: Could you run clang-format? The indentation looks off here.

Comment (andremarianiello) [reply opentracing/src/opentracing_directive.h]: Done.

**Comment by rnburn** [comment]
👍

**Comment by andremarianiello** [comment]
@rnburn When should I be expecting the next release?

**Pull request merged**